### PR TITLE
rustfmt

### PR DESCRIPTION
**Pull request by chocol4te**

### CODE REVIEW

File: src/radio.rs [hunk 1]
```diff
@@ -209,22 +243,34 @@ impl Transmitter for BleRadio {
...
+        header: advertising::Header,
+        channel: AdvertisingChannelIndex,
+    ) {
         let raw_header = header.to_u16();
-        self.tx_buf[0] = raw_header as u8;         // S0     = 8 bits (LSB)
-        self.tx_buf[1] = header.payload_length();  // Length = 6 bits
-        self.tx_buf[2] = 0;                        // S1     = 2 unused bits = 0
+        self.tx_buf[0] = raw_header as u8; // S0     = 8 bits (LSB)
+        self.tx_buf[1] = header.payload_length(); // Length = 6 bits
+        self.tx_buf[2] = 0; // S1     = 2 unused bits = 0
```
Comment (jonas-schievink) [inline src/radio.rs]: This now looks much worse than before

File: src/radio.rs [hunk 2]
```diff
@@ -69,28 +72,52 @@ impl BleRadio {
...
-            radio.crccnf.write(|w| w
-                .skipaddr().set_bit()   // skip address since only the S0, Length, S1 and Payload need CRC
-                .len().three()          // 3 Bytes = CRC24
+            radio.crccnf.write(
+                |w| {
+                    w.skipaddr()
+                        .set_bit() // skip address since only the S0, Length, S1 and Payload need CRC
+                        .len()
+                        .three()
+                }, // 3 Bytes = CRC24
```
Comment (jonas-schievink) [inline src/radio.rs]: rustfmt messed up the comments here too, best to move them before the method call

File: src/radio.rs [hunk 3]
```diff
@@ -115,17 +144,18 @@ impl BleRadio {
...
-        radio.shorts.write(|w| w
-            .ready_start().enabled()    // start transmission/recv immediately after ramp-up
-            .end_disable().enabled()    // disable radio when transmission/recv is done
+        radio.shorts.write(
+            |w| {
+                w.ready_start()
+                    .enabled() // start transmission/recv immediately after ramp-up
+                    .end_disable()
+                    .enabled()
+            }, // disable radio when transmission/recv is done
```
Comment (jonas-schievink) [inline src/radio.rs]: same here

File: Cargo.toml [hunk 4]
```diff
@@ -25,4 +25,5 @@ bitflags = "1.0.4"
 opt-level = "s"
 
 [profile.release]
-opt-level = "s"
+opt-level = 3
```
Comment (jonas-schievink) [inline Cargo.toml]: Does this actually make the image smaller? If so, we should do the same for debug builds (and a comment explaining what's going on).

Comment (chocol4te) [reply Cargo.toml]: My bad! I should've left that out. When I tested it made the image 34 bytes larger.

Comment (chocol4te) [reply Cargo.toml]: I meant to put the `debug = true` there, is a comment necessary explaining why?

Comment (jonas-schievink) [reply Cargo.toml]: Nah `debug = true` is fine as-is.

Comment (chocol4te) [reply Cargo.toml]: Fixed :)

**Pull request merged**